wemake_python_styleguide/visitors/ast/blocks.py

```diff
@@ -151,6 +153,25 @@ def visit_locals(self, node: Union[AnyAssign, ast.arg]) -> None:
 
     # Utils:
 
+    def _is_function_overload(self, node: ast.AST):
+        if isinstance(node, self._ast_func):
+            _scope = BlockScope(node)
+            for decorator in node.decorator_list:
+                if (
```

Comment:
This solution is overly complicated. We do not need to check where the import is from. We don't care about the type. The only thing we care about is the name.

So, this can be changed to the following: 

```python
if node_to_string(decorator) in {'overload', 'typing.overload'}: ...
```